tests/pytests/test_dipoles.py

```diff
@@ -45,3 +45,31 @@ def test_dipole(inp):
     analytic_dipole = wfn.variable(inp['varname'] + " DIPOLE")
 
     assert compare_values(findif_dipole, analytic_dipole, 5, "findif vs. analytic dipole")
+
+@ pytest.mark.slow
+@pytest.mark.parametrize("inp", [
+    pytest.param({'name': 'ccsd', 'options': {'opdm_relax': 'true'}, 'varname': 'CCSD'}, id='ccsd')
```

Comment:
good test. why not merge it with `test_dipole` since all but the reference repeated?

ah, molecule. can do `mol = h2o_doublet if "reference" not in inp["options"] else h2o_singlet`

Reply:
So, define both `h2o_doublet` and `h2o_singlet` inside the test?

Reply:
yes, and then pass `properties(..., molecule=mol)`

Reply:
actually, `mol = h2o_singlet if inp["options"].get("reference", "rhf") == "rhf" else h2o_doublet` would be more robust